source/core/src/main/com/csse3200/game/components/player/KeyboardPlayerInputComponent.java

```diff
@@ -92,6 +92,15 @@ public boolean keyDown(int keycode) {
       case Keys.I:
         entity.getEvents().trigger("statsInfo");
         return true;
+      case Keys.NUM_1: // TEMPORARY: Press 1 to spawn the land boss
+        entity.getEvents().trigger("defeatLandBoss");
```

Comment:
these events are reserved for quest completion upon boss death, see other comment